src/components/GeoMap.js

```diff
@@ -473,7 +472,7 @@ export default class GeoMap {
         } else {
           const originalValueRaw = d.rollupValue[attrKey];
           const originalValue = (attrValue.transform) ?
-              attrValue.transform(originalValueRaw) : originalValueRaw;
+              attrValue.transform(originalValueRaw, this.maptable.rawData) : originalValueRaw;
```

Comment:
Do you have any example on how you calculate the percentile? Maybe it can be interesting to see its application in the example folder